modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -194,24 +194,86 @@
 }
 
 // Responsive mode
-// This code needs to be updated in the Responsive Tabs section task
-@media (max-width: $screen-mobile-max) {
-	.e-n-tabs {
+@mixin hide-normal-tabs-($device) {
+	.e-n-tabs-#{$device} {
 
 		.e-n-tabs-heading {
 			display: none;
 		}
 	}
 }
 
-@media (min-width: $screen-mobile-next) {
-	.elementor-widget-n-tabs {
+@mixin hide-collapse-tabs-($device) {
+	.e-n-tabs-#{$device} {
+
+		.e-collapse {
+			display: none;
+		}
+	}
+}
 
-		.e-n-tabs {
-			
-			.e-collapse {
-				display: none;
+@mixin add-gap-and-distance-($device) {
```

Comment:
We can remove the mixin, because we don't need any breakpoints for this. 
We have already hidden the `.e-collapse` elements using the breakpoints already.

The disadvantage of these mixins in combination with additional breakpoints, is that they bloat the code. We get code duplication inside the css. And we can avoid that.

Reply:
mixin removed